src/docfx/lib/ParallelUtility.cs

```diff
@@ -25,7 +25,14 @@ public static void ForEach<T>(LogScope scope, ErrorBuilder errors, IEnumerable<T
             }
             catch
             {
-                Console.WriteLine($"Error processing '{item}'");
+                if (item is string str)
+                {
+                    Console.WriteLine($"Error processing '{UrlUtility.SanitizeUrl(str)}'");
```

Comment:
nit: no need for if?
```suggestion
                    Console.WriteLine($"Error processing '{UrlUtility.SanitizeUrl(str?.ToString())}'");
```